src/applications/personalization/dashboard/utils/timezone.js

```diff
@@ -32,3 +33,109 @@ export const getCCTimeZone = appointment => {
   const zoneSplit = appointment.attributes?.timeZone.split(' ');
   return stripDST(zoneSplit[1]);
 };
+
+export function getTimezoneByFacilityId(id) {
+  if (!id) {
+    return null;
+  }
+
+  if (timezones[id]) {
+    return timezones[id];
+  }
+
+  return timezones[id.substr(0, 3)];
```

Comment:
I dont think this will work as it is right now, because the timezones.json file from VAOS is a singular object, and the timezones.json from the dashboard codebase is an array instead.

https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/vaos/utils/timezones.json